digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/package.scala

```diff
@@ -32,6 +32,8 @@ package object viite {
 
   val MinDistanceForGeometryUpdate = 0.5
 
+  val MaxThreshHoldDistance = 2
```

Comment:
Fix speling mistake:
MaxThreshHoldDistance -> MaxThresholdDistance

Reply:
(This kind of minor things can be fixed to the pull request that is for the master branch.)